source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEvaluatorEmployeesCommand.cs

```diff
@@ -50,7 +51,16 @@ public override List<EmployeeEvaluationDTO> ExecuteWithResult()
                     State = EmployeeEvaluationDTO.GetEvaluationState(e.AutoEvaluationDone, e.ResponsibleEvaluationDone, e.CompanyEvaluationDone, e.OpenToDevolution, e.Finished),
                     Id = e.Id,
                     TemplateId = e.TemplateId,
-                    IsEditable = getEvaluationEditable(e)
+                    IsEditable = getEvaluationEditable(e),
+                    PosibleRevertActions = RevertEvaluationActionsHelper.GetPosibleRevertActions(
+                        e.Finished,
+                        e.OpenToDevolution,
+                        e.CompanyEvaluationDone,
+                        e.ResponsibleEvaluationDone,
+                        e.AutoEvaluationDone,
+                        e.EvaluatorEvaluationDone)
```

Comment:
Since there are more than a _Evaluator Evaluation_ it is not clear what it means, what do you think about `AllEvaluatorsEvaluationDone` or `AnyEvaluatorsEvaluationDone`?